src/screens/LocalConfirmNotificationCodeScreen.tsx

```diff
@@ -39,11 +39,8 @@ class LocalConfirmNotificationCodeScreen extends PureComponent<Props, State> {
     numberAttempt: 0,
   };
 
-  componentWillUnmount() {
```

Comment:
I think this is needed because the error will stay after clicking the back button